.eslintrc.js

```diff
@@ -77,5 +77,6 @@ module.exports = {
     "@typescript-eslint/default-param-last": 0,
     "@typescript-eslint/no-explicit-any": 0,
     "@typescript-eslint/no-children-prop": 0,
+    "@typescript-eslint/no-unused-expressions": 0,
```

Comment:
Oh!  Yeah, I had been playing with this setting because it was complaining about some legitimate code, but it dien't help in the end anyway.  We can just remove it.